tests/Util/TestHandler.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of sensiolabs-de/storyblok-bundle.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Storyblok\Bundle\Tests\Util;
```

Comment:
Should be under:
```suggestion
namespace SensioLabs\Storyblok\Bundle\Tests\Double;
```
namespace

I personally would name it "ConfigurableHandler", as you configure it via actor.

Reply:
How would you name the one which throws an exception then?